media-gfx/gifsicle/gifsicle-1.96.recipe

```diff
@@ -31,6 +31,9 @@ PROVIDES="
 REQUIRES="
 	haiku$secondaryArchSuffix
 	"
+REPLACES="
+	gifsicle
+	"
```

Comment:
This needs to go into an `if [ -n "$secondaryArchSuffix" ]` block, otherwise it would replace itself. Not sure if that works ...

Reply:
In this case it's more important to check 64 bit ;-)

Reply:
How's that, it's primary arch already on 64bit, so this will replace the already existing one already without this change?

Reply:
On 64 bit, the package name is the same, so the REPLACES isn't needed. If you add it unconditionally, the package would replace itself, which doesn't make sense. The REPLACES is only needed for 32 bit, where `gifsicle_x86` should replace old versions of `gifsicle`.

Reply:
This should do it I think, checked on both arch's. :)